src/02-table-tests/index.test.ts

```diff
@@ -1,17 +1,27 @@
 // Uncomment the code below and write your tests
-/* import {  simpleCalculator, Action } from './index';
+import { simpleCalculator, Action } from "./index";
 
 const testCases = [
-    { a: 1, b: 2, action: Action.Add, expected: 3 },
-    { a: 2, b: 2, action: Action.Add, expected: 4 },
-    { a: 3, b: 2, action: Action.Add, expected: 5 },
-    // continue cases for other actions    
-]; */
+  { a: 5, b: 7, action: Action.Add, expected: 12 },
+  { a: 5, b: 7, action: Action.Subtract, expected: -2 },
+  { a: 5, b: 7, action: Action.Multiply, expected: 35 },
+  { a: 6, b: 3, action: Action.Exponentiate, expected: 216 },
+  { a: 6, b: 3, action: Action.Divide, expected: 2 },
+  { a: 6, b: null, action: Action.Exponentiate, expected: null },
+  { a: 6, b: null, action: Action, expected: null }
```

Comment:
It seems like 3 should be here because you are supposed to check the wrong action, not an invalid argument. As it was in simple tests.
```suggestion
  { a: 6, b: 3, action: Action, expected: null }
```